.travis.yml

```diff
@@ -1,3 +1,8 @@
 language: csharp
+dotnet: 3.1.301
+
+env:
+  - MSBuildSDKsPath=/usr/share/dotnet/sdk/3.1.301/Sdks
```

Comment:
Ah, this error. it was supposed to be fixed following https://github.com/dotnet/sdk/issues/9632 and the related bugs though 🤔  But oh well